### DISCUSSION TITLE
ru-RU/contribution/feedback

### DISCUSSION THREAD

**Discussion by giscus** [Gzh0821/pvzg_site#15] [Announcements]
# ru-RU/contribution/feedback

Совет Если вам нравится этот веб-сайт и игры, вы можете собрать/пометить этот веб-сайт на складе Github Если вы хотите сообщить об ошибках в игре и внести предложения, пожалуйст...

https://www.pvzge.com/ru-RU/contribution/feedback.html

**Comment by NoobikRoma** [comment]
I LOVE THIS GAME!

**Reply by NameZeroYT** [reply]
РОМА? ТЫ ЧЁ ТУТ ДЕЛАЕШЬ?

**Reply by NoobikRoma** [reply]
I don't know you

**Comment by IggyZetekov** [comment]
EN: Hello, Gardendless Team! My name is Iggy, I from Russia. I want to thank you very much for making the dream of all those who have been waiting for PvZ 2 on PC come true! I wish you further development of the project and get an inordinate amount of pleasure from it. I am also writing with a proposal to add to the sandbox mode the ability to freely place water tiles, gravestones, minecarts, ice floes, trap tiles and other grid item from future updates on the lawn (Like power tiles, potions). And once again, I wish you success in your development! ;)

CH: 你好，Gardendless团队！ 我叫伊吉，来自俄罗斯。 我要感谢你让所有那些在PC上等待PvZ2的人的梦想成真！ 我希望你进一步发展这个项目，并从中获得过多的乐趣。 我也在写一个建议，以增加沙箱模式的能力，自由地放置水砖，墓碑，矿车，浮冰，陷阱瓷砖和其他网格项目从草坪上的未来更新（如电源瓷砖，药水）。 再次祝你发展成功！ ;)

**Comment by NameZeroYT** [comment]
Это моя идея была!

**Reply by IggyZetekov** [reply]
Не кипяшуй. Я ж не знал, что это твоя идея была.

**Reply by IggyZetekov** [reply]
Тем более, не мы одни этого хотим

**Comment by NameZeroYT** [comment]
Hello, Gardendless Dev Team!
I know that you probably going to ignore it by now, just here is my concept for replacing the plant leveling. I know that many PvZ 2 fangames and mods replace the OG leveling system. I suggest making only 5 levels, each level boosting plant's damage by 10% and it's health by 20%. On level 5, which is called "Plants Mastery", Plant damage is boosted by an additional 5% and its cost is reduced by 25 suns. Each level's cost is presented at the end. Also, I can suggest unlocking Golden Costume for a plant that reached Mastery.

Level 1: free, +0% damage, +0% health (basic level).
Level 2: 500 coins and 5 gems, +10% damage, +20% health.
Level 3: 1000 coins and 5 gems, +10% damage, +20% health.
Level 4: 1500 coins and 10 gems, +10% damage,  +20% health.
Level 5: 2500 coins and 10 gems, +5% damage, -25 sun cost.

Additional info: health is not boosted for plants like Chili Bean and Hypno-shroom.

**Comment by Duken228** [comment]
Привет , когда на новой версии пофиксят баг что на 11 уровне в пиратской бухте уровень пройти невозможно. даже если вечно делать x4 пройти нереально (на старой версии 4.0 такого не было)

**Reply by sua91ikj-crypto** [reply]
Привет команда разработчиков , у меня после перехода на версию 0.12.1 пропали пеньяты

**Comment by sua91ikj-crypto** [comment]
Привет команда разработчиков , у меня после перехода на версию 0.12.1 пропали пеньяты
